I removed all of the slots from this board

coreyker opened the pull request:
As outline on the lines forum [1], I found that the slots on this board pose an issue for manufacturability with some fab houses. The slots were created by drawing an outline on the milling layer; however, this may be problematic because Eagle’s DRC ignores this layer [2]:

"Unlike the Dimension layer, which can interact with your design rules, the Milling layer does not."

This means that copper pours may go right through a slot, and create a potential short [3]:

"When using Slots with a 4 layer PCB, you MUST manually verify internal layers. Eagle’s DRC does not respect any method of making slots, and copper pours or auto-routed traces will short to the slot."

My solution was to remove all of the slots from the PCB and only use normal pads (which respect Eagle’s DRC). I didn't have any manufacturing issues with my slotless boards.

It might make sense -- if you decide my changes are relevant -- to keep them on a separate "slotless" branch, as I have done in my fork; that way the original design and the modified design are available to any interested parties.

[1] https://llllllll.co/t/teletype-hardware-open/27242/138?u=frankchannel
[2] https://www.autodesk.com/products/eagle/blog/every-layer-explained-autodesk-eagle
[3] https://docs.oshpark.com/design-tools/eagle/cutouts-and-slots

coreyker committed:
An alternative version with all slots from the board (this may be easier for some fab houses to manufacture)

tehn commented:
thank you! could you attach some screenshots of the changes? those slotted jacks are really a pain--- i want to make sure this modification doesn't compromise the build.

coreyker commented:
Understandable. It's not like software, where you can run some simple tests to see if a change is breaking or not. I've attached two PDF printouts comparing the boards. If you flip back and forth between the two PDFs in an image editor, you can see the changes pretty clearly. The parts that I have changed are: 
1. all the jacks
2. the potentiometer, and;
3. the USB connector. 

I simply removed the slots from these parts and replaced them with pads. I only had to re-route 1 trace: the one going from VBUS through C7 to ground. I also made the board about 1mm longer so that the pads at the bottom didn't get cutoff (there was a DRC warning about this, I believe).

[teletype-180824.pdf](https://github.com/monome/teletype-hardware/files/4779788/teletype-180824.pdf)
[teletype-180824-slotless.pdf](https://github.com/monome/teletype-hardware/files/4779789/teletype-180824-slotless.pdf)

I have built and tested 1 board, and it seems to work fine. I would be happy to mail you off a board if you'd like. Maybe you have a testing jig/setup? (The boards I had fabbed have most of the passive components pre-soldered, but none of the ICs or through hole components).

tehn commented:
i see, thank you for the pdfs.

i've done similar round-holes when i'm quickly prototyping, but in my experience part alignment is much better with slots when doing production runs. i'd like to keep the existing files as-is, but i do think it'd be good to host your edits here, and add a note to the readme regarding possible problems with pcb manufacturers--- since they now is a rise of no-engineer-checking-your-files services.

could you edit or resubmit the PR? just adding `-no-slots` to the filenames would be adequate perhaps?

coreyker commented:
Makes sense, and I agree that your original files should remain as they are. I moved my changes to new files with the `-no-slots` postfix, and rebased my commit.
